### DISCUSSION TITLE
Feature Request: Ability to move Github Issue to a different repository within the same organization

### DISCUSSION THREAD

**Discussion by JLLeitschuh** [community/community#22597] [New to GitHub]
<p>EDIT: This is now an official feature! <a href="https://blog.github.com/changelog/2018-10-30-issue-transfer/" rel="nofollow noopener">https://blog.github.com/changelog/2018-10-30-issue-transfer/</a></p>
<p>There are some organizations that have several repositories with various different peices of software.</p>
<p>Sometimes these organizations have users are new to Github and put issues under the wrong repository.</p>
<p>It would be nice if there were mechanisims in place for maintainers to move issues between organizations.</p>
<p>ZenHub offeres such a mechanism but it’s a hack on top of the Github API and all comments in the issue being moved get compressed into one comment instead of staying as “owned” by the person who opened them.</p>
<p><a href="https://www.zenhub.com/product" rel="nofollow noopener">https://www.zenhub.com/product</a></p>
<p>It would be nice if this could be offered as a seemless feature within the Github UI and API.</p>
<p>There’s another unoficial mover:<br>
<a href="https://github-issue-mover.appspot.com" rel="nofollow noopener">https://github-issue-mover.appspot.com</a></p>

**Comment by nadiajoyce** [comment]
<p>Hi <a class="mention" href="/jlleitschuh">@jlleitschuh</a>,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>

**Comment by JLLeitschuh** [comment, marked as answer]
<p>Wow! Great response time! Nice job GitHub team!!!</p>
<p><a href="https://blog.github.com/changelog/2018-10-30-issue-transfer/" rel="nofollow noopener">https://blog.github.com/changelog/2018-10-30-issue-transfer/</a></p>
